src/main/java/com/github/platymemo/alaskanativecraft/item/LabradorTeaItem.java

```diff
@@ -0,0 +1,113 @@
+package com.github.platymemo.alaskanativecraft.item;
+
+import net.minecraft.advancement.criterion.Criteria;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.entity.effect.StatusEffect;
+import net.minecraft.entity.effect.StatusEffectInstance;
+import net.minecraft.entity.effect.StatusEffects;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.ItemUsage;
+import net.minecraft.item.Items;
+import net.minecraft.server.network.ServerPlayerEntity;
+import net.minecraft.sound.SoundEvent;
+import net.minecraft.sound.SoundEvents;
+import net.minecraft.util.Hand;
+import net.minecraft.util.TypedActionResult;
+import net.minecraft.util.UseAction;
+import net.minecraft.world.World;
+
+import java.util.List;
+
+/**
+ * original code from bodacious berries by ix0rai
+ */
+public class LabradorTeaItem extends Item {
+    private final List<StatusEffect> negativeEffects = List.of(
+            StatusEffects.POISON,
+            StatusEffects.BLINDNESS,
+            StatusEffects.NAUSEA,
+            StatusEffects.HUNGER,
+            StatusEffects.WEAKNESS,
+            StatusEffects.SLOWNESS,
+            StatusEffects.MINING_FATIGUE,
+            StatusEffects.BAD_OMEN,
+            StatusEffects.WITHER,
+            StatusEffects.UNLUCK
+    );
```

Comment:
I think there should be a more extensible way to do this: something data-driven perhaps. I know the Akutaq certainly could use that as a revamp. There is StatusEffectCategory which holds whether an effect is beneficial, neutral, or harmful. Perhaps that is a good basis. Negative effects can be wiped by the labrador tea, and positive effects can be randomly queried by the akutaq.